pkg/ovs/openflow/interfaces.go

```diff
@@ -128,16 +127,22 @@ type Bridge interface {
 // TableStatus represents the status of a specific flow table. The status is useful for debugging.
 type TableStatus struct {
 	ID         uint      `json:"id"`
+	Name       string    `json:"name"`
 	FlowCount  uint      `json:"flowCount"`
 	UpdateTime time.Time `json:"updateTime"`
 }
 
 type Table interface {
-	GetID() TableIDType
+	GetID() uint8
+	GetName() string
 	BuildFlow(priority uint16) FlowBuilder
 	GetMissAction() MissActionType
 	Status() TableStatus
-	GetNext() TableIDType
+	GetNext() uint8
+	SetNext(next uint8)
+	SetMissAction(action MissActionType)
+	SetBridge(bridge Bridge)
```

Comment:
Where are these SetXXX methods used?

Reply:
Each one of these functions sets one property. I want to keep them for single property update case. They are not used in this PR, but I think they are usable in the future flexible pipeline. For example, we could create table first, and then change the next ID along with pipeline is changed.